### PR TITLE
Can ignore this :v: 

### PR DESCRIPTION
**Pull request by VoChiDanh**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# User-specific stuff
```
Comment (dmccoystephenson) [inline .gitignore:1]: Is this necessary? Can you please remove this from the tracked files if it is not necessary?

File: .idea/.gitignore [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (dmccoystephenson) [inline .idea/.gitignore:1]: Is this necessary? Can you please remove this from the tracked files if it is not necessary?

File: .idea/compiler.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dmccoystephenson) [inline .idea/compiler.xml:1]: Is this necessary? Can you please remove this from the tracked files if it is not necessary?

File: .idea/encodings.xml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dmccoystephenson) [inline .idea/encodings.xml:1]: Is this necessary? Can you please remove this from the tracked files if it is not necessary?

**Review by dmccoystephenson — CHANGES_REQUESTED** [review]
The changes look good! I left a few comments concerning whether or not some files were necessary to add to the tracked files.

File: .gitignore (hunk 1 above)
Comment (VoChiDanh) [reply .gitignore:1]: This is just a small file to help you not push unnecessary folders and files, i think so

File: .idea/.gitignore (hunk 2 above)
Comment (VoChiDanh) [reply .idea/.gitignore:1]: This is just a small file to help you not push unnecessary folders and files, i think so

Comment (VoChiDanh) [reply .idea/.gitignore:1]: This file came from gitignore, no big deal

File: .idea/compiler.xml (hunk 3 above)
Comment (VoChiDanh) [reply .idea/compiler.xml:1]: This file came from gitignore, no big deal

File: .idea/encodings.xml (hunk 4 above)
Comment (VoChiDanh) [reply .idea/encodings.xml:1]: This file came from gitignore, no big deal

**Review by dmccoystephenson — APPROVED** [review]

**Comment by Mr-Deej** [comment]
this possibly closed #1284 will need feed back from players on 1.16 and java 8 or 11

**Pull request merged**